agent/engine/task_manager.go

```diff
@@ -61,7 +60,7 @@ const (
 	maxStoppedWaitTimes                      = 72 * time.Hour / stoppedSentWaitInterval
 	taskUnableToTransitionToStoppedReason    = "TaskStateError: Agent could not progress task's state to stopped"
 	// unstage retries are ultimately limited by successful unstage or by the unstageVolumeTimeout
-	unstageVolumeTimeout = 30 * time.Second
+	unstageVolumeTimeout = 600 * time.Second
```

Comment:
This is a pretty big jump in the unstage timeout duration. I know that some processes can vary between OS types/platforms, could this be scoped down for just windows without affecting for all?

Reply:
Yes, we will refactor this to just affect windows and not linux.